pkgs/tools/misc/bat-extras/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchFromGitHub, bash, less, makeWrapper, ncurses, bat, ripgrep
+, withShFmt ? true, shfmt
+, withPrettier ? true, nodePackages
+, withClangTools ? true, clang-tools
+, withRustFmt ? true, rustfmt
+, withEntr ? true, entr
+}:
+
+stdenv.mkDerivation rec {
+  pname   = "bat-extras";
+  version = "20200408";
+
+  src = fetchFromGitHub {
+    owner  = "eth-p";
+    repo   = pname;
+    rev    = "v${version}";
+    sha256 = "184d5rwasfpgbj2k98alg3wy8jmzna2dgfik98w2a297ky67s51v";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ bash makeWrapper ] ++ stdenv.lib.optional withShFmt shfmt;
+
+  phases = [ "unpackPhase" "buildPhase" "installPhase" ];
+
+  buildPhase = ''
+    patchShebangs test.sh
+    patchShebangs .test-framework/bin/best.sh
+
+    chmod u+x .test-framework/lib/print_util.sh
+    wrapProgram .test-framework/lib/print_util.sh \
+      --prefix PATH : "${ncurses}/bin" \
```

Comment:
This seems wrong. This is a library file that's sourced, not an executable. Wrapping it like this means the act of sourcing it will modify the path. Instead we should make `${ncurses}/bin` available either in a wrapper of `.test-framework/bin/best.sh`, or of `test.sh` itself, or just globally during the build.